cfn/README-GD.md

```diff
@@ -59,125 +67,110 @@ An example of a successful response is:
 ```
+
 An example of a successful response is:
 
 ```
 @{access_key_id=712c0b413eef41f6; secret_key=1234567890b3eea8880d292fb31aa96902242a076d3d0e320cc036eb51bf25ad}
 ```
 
-**Important:** If the command returns an error about not having the necessary role, please verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
-
 Make a note of the `access_key_id` and `secret_key` values, which you need to deploy the CloudFormation template to your AWS account.
 
-**Note:** Each user can create only five access keys. If a "limit exceeded" response appears, you must delete one or more access keys before you can create new keys.
+**Necessary role error**
 
-**Type the following command to list access keys:**
-```
-Invoke-RestMethod -Method Get -Headers @{"x-aims-auth-token"=$AL_TOKEN} -Uri https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys
-```
+If the command returns an error about not having the necessary role, please verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
 
-**Use the selected access_key_id in the following command to delete the key:**
+**"Limit exceeded" error**
 
-```
-Invoke-RestMethod -Method Delete -Headers @{"x-aims-auth-token"=$AL_TOKEN} -Uri https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys/<ACCESS_KEY_ID_HERE>
-```
+Each user can create only five access keys. If a "limit exceeded" response appears, you must delete one or more access keys before you can create new keys.
 
+1. Type the following command to list access keys:
```

Comment:
the same issue as here - https://github.com/alertlogic/cwe-collector/pull/25/files#r163519306, please fix number of spaces in markdown to get ordered list